### PR TITLE
Readme edits

### PR DESCRIPTION
**Pull request by gordonzhang17**
This is NOT ready for a merge, just want feedback for now.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,59 +1,124 @@
...
+**Node.js/Express.js**
+
+We used these two for building the backend. Our frontend interacts with 
+the endpoints and the backend interacts with the database. Database operations (GET/POST/DELETE/PATCH) are supported 
+depending on the collection.
+
+**MongoDB** 
+
+We included Mongoose as well. We used MongoDB Atlas as our backend cloud database storage of information. 
+We had collections including parks, users, events, park comments, and event comments.
```
Comment (lydia-zheng) [inline README.md:77]: Do we want to mention facilities and features too?

File: README.md [hunk 2]
```diff
@@ -1,59 +1,124 @@
...
+frontend/backend functionality for usersZ
+- Ensure local storage in browser stores redux data so refreshing page persists user information
+
+Claire Anderson
+- Came up with the original concept for the project
+- Implemented the modal system, routing, and the calendar page in the front end
+- Completed the end-to-end implementation of filtering parks by parameters as well as their location, 
+and the editing and deleting of events
+
+Philippe Chapdelaine
```
Comment (lydia-zheng) [inline README.md:120]: Phil seems to be missing his part?

File: README.md (hunk 1 above)
Comment (gordonzhang17) [reply README.md:77]: I didn't want to list every single collection, these are the most important

File: README.md (hunk 2 above)
Comment (gordonzhang17) [reply README.md:120]: yes im adding it now

File: README.md [hunk 3]
```diff
@@ -1,60 +1,130 @@
 # ParkHangs
 
-ParkHangs is an application that allows users to view details for all official Vancouver Parks. Users are able to get detailed information about a specific park, save favourite parks, create and integrate events to their Google Calendar, as well as write comments about a park and events.
+ParkHangs is an all-in-one platform that allows users interact with all official Vancouver, British Columbia Parks. Users can get detailed information about a specific park, save favourite parks (if they are logged in), create and integrate events to their own calendar application, as well as write comments about parks and events.
```
Comment (cmaija) [inline README.md:3]: all City of Vancouver Public Parks

**Pull request closed without merging**